src/main/java/model/User.java

```diff
@@ -1,4 +1,43 @@
 package model;
 
 public class User {
+
+    private int id;
+    private String name;
+    private boolean blocked;
+
+    public User(int id, String name, boolean blocked) {
+        this.id = id;
+        this.name = name;
+        this.blocked = blocked;
+    }
+
+    public User() {
+        super();
```

Comment:
??

Reply:
I don't know why, but read JSON file does not work without default constructor.
Here is the solution:
https://facingissuesonit.com/2019/07/17/com-fasterxml-jackson-databind-exc-invaliddefinitionexception-cannot-construct-instance-of-xyz-no-creators-like-default-construct-exist-cannot-deserialize-from-object-value-no-delega/